### PR TITLE
Added System.Types to uses 

### PR DESCRIPTION
**Pull request by MHumm**
In order to suppress warnings about impossible optimization/inlining

### CODE REVIEW

File: jvcl/run/JvAni.pas [hunk 1]
```diff
@@ -40,6 +40,9 @@ interface
   {$IFDEF UNITVERSIONING}
   JclUnitVersioning,
   {$ENDIF UNITVERSIONING}
+  {$IFDEF HAS_UNIT_TYPES}
```
Comment (obones) [inline jvcl/run/JvAni.pas]: No need for the IFDEF here, Types was introduced by Delphi 6 and we don't support anything earlier than Delphi 6

File: jvcl/run/JvAni.pas [hunk 2]
```diff
@@ -40,6 +40,9 @@ interface
...
+  System.Types,
```
Comment (obones) [inline jvcl/run/JvAni.pas]: Do not prefix, the namespace support was added later on.
Basically, Types could have been simply added at the end of the next line

Comment (MHumm) [reply jvcl/run/JvAni.pas]: Ok, good to know, so I changed it.

**Comment by obones** [comment]
Could you please pull the "upstream" changes into the master branch of your repository?
This would make the occurrence of merge conflicts less likely

https://help.github.com/en/articles/syncing-a-fork

**Comment by MHumm** [comment]
Ok, will pull changes more often. Git is still new to me and I'm not using it besides for helping here.

**Pull request merged**